src/test/java/Runners/Run.java

```diff
@@ -7,7 +7,6 @@
         features = {
                 "C:\\Users\\rudi\\IdeaProjects\\CucumberTestNG\\resources\\CourseWork.feature"},
```

Comment:
features = "src/test/resources/Features",
        glue = "StepDefinitions",
where Features is folder where feature files are located and StepDefinitions - folder where Steps class is located

Reply:
also, pay attention to the feature file should be in the **test**/resources